code/modules/projectiles/guns/projectile/boltaction.dm

```diff
@@ -2,29 +2,29 @@
 
 /obj/item/weapon/gun/projectile/shotgun/pump/rifle
 	name = "bolt action rifle"
-	desc = "A reproduction of an almost ancient weapon design from the early 20th century. It's still popular among hunters and collectors due to its reliability. Uses 7.62mm rounds."
+	desc = "A reproduction of an almost ancient weapon design from the early 20th century. It's still popular among hunters and collectors due to its reliability. Uses 5.56mm rounds."
```

Comment:
This is a terrible idea and I don't know why you're doing this. The whole point of the bolt action rifle is it has the same ROF as a shotgun, but is easier to reload in exchange for having a much shittier magazine. If anything, this deserves to stay as 7.62 even with the ammo's buff. It makes the weapon a lot less useless.

This also makes no sense considering the only gun available to players that is a bolt rifle are either the Mosins from cargo, or the competition shooting rifles in security.

To balance this, if you think it's too easy to get Mosins via cargo, adjust the point cost, or adjust how many you get with each crate.

In all, I am strongly against changing the bolt action rifle to 5.56, even with the buff to damage. It makes no sense to lore or balance.